### DISCUSSION TITLE
Свайпы, масштабирование вложенных фотографий в новостях

### DISCUSSION THREAD

**Discussion by ValeryVerkhoturov** [0niel/university-app#217] [Идеи]
**Проблема**
В разделе Новости в некоторых статьях под основным текстом есть сборники фотографий. Есть возможность нажать на фотографию для показа на весь экран, масштабировать ее без ограничений, выйти из показа на полный экран по кнопке слева сверху.

**Решение**
1. Добавить свайпы влево/вправо для листания фотографий;
2. Добавить свайпы вверх/вниз для выхода из показа фотографии на весь экран;
3. Ограничить масштабирование.

**Comment by Dragonprod** [comment]
#353 
Технические ограничения не позволяют реализовать выход из показа фотографии на весь экран. В будущем реализуем.
